twake/backend/node/src/services/user/services/companies/service.ts

```diff
@@ -60,6 +61,15 @@ export class CompanyService implements CompaniesServiceAPI {
     return this.externalCompanyRepository.findOne(pk);
   }
 
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  @RealtimeSaved<Company>((company, _context) => {
+    return [
+      {
+        room: `/company/${company.id}`,
+        resource: company,
+      },
+    ];
+  })
```

Comment:
This will not work, the updateCompany function must contain a "context" as last argument and should return a SaveResult object.

Reply:
Also we need to add a RealtimeSaved inside removeUserFromCompany and setUserRole (to make companies appear or disappear)

Reply:
Updated, could you check if it's good please? :)